dspace/config/spring/api/discovery.xml

```diff
@@ -61,6 +61,11 @@
                 <entry key="publicationVolume" value-ref="publicationVolume"/>
                 <entry key="periodical" value-ref="periodical"/>
                 <entry key="project" value-ref="project"/>
+                <!-- OpenAIRE4 guidelines - search for a creator that can be a Person
+                     or an OrgUnit -->
+                <entry key="openAIRECreatorConfiguration" value-ref="openAIREAuthor"/>
+                <!-- OpenAIRE4 guidelines - search for an OrgUnit -->
+                <entry key="openAIREFundingAgencyConfiguration" value-ref="openAIREFundingAgency"/>
```

Comment:
The comment above this line is slightly incorrect. It does NOT appear to be a simple "orgUnit" search (if so, you could just use the `organization` configuration that already exists).  Instead, this is searching for `OrgUnit` that have `dc.type=FundingOrganization`